templates/search/blocks/search_filters.html

```diff
@@ -135,7 +135,7 @@ <h4 class="search-filters__accordion-section-heading" id="record_opening_date">R
             {% endif %}
         </div>
 
-        {{ 'catalogue-filters-form'|include_hidden_fields:form }}
+        {% render_form_fields_as_hidden form "filter_keyword collection topic level closure opening_start_date opening_end_date created_start_date created_end_date catalogue_source held_by type country location" "created_start_date created_end_date opening_start_date opening_end_date" %}
```

Comment:
Like I explained earlier, the tag should have an `include` option for cases like this, allowing you supply `"q sort_by"` here, and not have to update this list here to account for every field on every form.

Reply:
@ababic 
the original code is `{{ 'filter_keyword collection topic level closure opening_start_date opening_end_date catalogue_source held_by type country location'|include_hidden_fields:form }}` which uses `@register.filter`
As you've suggested I've updated it to `@register.simple_tag` with the new name `render_form_fields_as_hidden`

The original bit handles the form fields that need to be included by excluding the ones from the list. This avoids some manual update when a new form field is introduced.

Why do we want to re-write to have the `include` param?